### DISCUSSION TITLE
Statiq.Web + CleanBlog theme:  Blog posts produced locally, but not in GitHub pages(?)

### DISCUSSION THREAD

**Discussion by jcoo092** [statiqdev/Discussions#202] [Q&A]
Hello Mr Glick (and anybody else) 👋 

I have a thoroughly confusing issue to which I can't find a solution.  Namely, that when I build and run Statiq.Web to generate a static site locally with `dotnet run -c Release -- preview`, the blog post files are processed and added to the produced website, which I can observe by checking the version served on localhost.  When I run the GitHub action to deploy to GitHub pages, however, the blog post files seem to be skipped over for some reason in the processing, and thus they aren't output and the resulting site doesn't include them.  I can't for the life of me work out what the difference is that is causing this, though.

You can see the repo in question [here](https://github.com/jcoo092/jcoo092.github.io/commit/e997414a59a39dcf773047cac59f51d17fa512b4) (that points to the current HEAD as of the time of writing).  I even set the log level to `Debug` to see if that would generate further insights, but I couldn't find anything relevant.  The [end of the Inputs phase](https://github.com/jcoo092/jcoo092.github.io/actions/runs/5515042144/jobs/10054957481#step:4:587) just says that there were two fewer output documents than input documents.  The numbers match, however, on the local run.

Do you have any ideas what might be causing the difference in behaviour?  Or how I could try to debug this further?  Or indeed how I might resolve the issue?

**Comment by AlexHedley** [comment]
Were you taking the output generated from the build and deploying that to GitHub pages?

Example [main.yml](https://github.com/alex-hedley/statiqweb-example/blob/main/.github/workflows/main.yml)

**Reply by jcoo092** [reply]
I don't totally remember any more, to be honest.  I switched over to using Hugo (not that I _really_ wanted to) when I couldn't figure out a solution to the problem.

If I recall correctly, I was indeed taking the output of the build and publishing it to the site properly—everything else was published fine.  The build process just seemed to ignore the input blog files (such as they were at the time) completely, so there was never any output for them.
